### ISSUE TITLE
Cannot compare value of date column with LocalDate

### ISSUE THREAD

**Issue by rherrmann** [assertj/assertj-db#69]
Given a table with a date column, the following attempt to compare the value of a date column of a specific row with a `LocalDate` fails:

```java
Assertions.assertThat(someTable).row(0).value("date_column").isEqualTo(LocalDate.of(2020, 4, 6));
```

The failure message is:
```
java.lang.AssertionError: [Value at index 3 (column name : DATE_COLUMN) of Row at index 0 of 'select ...' request] 
Expecting:
  <2020-04-06T00:00:00.000>
to be of class
  <java.time.LocalDate>
but was of class
  <java.sql.Date>
```

It seems possible to use a string or `java.sql.Date` instead. Is there a reason why `isEqualTo(LocalDate)` is not supported?

**Comment by VanRoy** [comment]
@rherrmann Can you tell me which DB you use for the test and the table schema ?

**Comment by rherrmann** [comment]
The test runs against Postgres 12.2.

**Comment by VanRoy** [comment]
Thanks @rherrmann , Can you tell me what is the type of the column in DB ?

**Comment by rherrmann** [comment]
The type is just `date`.

**Comment by VanRoy** [comment]
@rherrmann Thanks a lot, I will investigate.

**Comment by VanRoy** [comment]
@rherrmann Unfortunately LocalDate is not available on assertion for now. JSR 310 support is planned for the next version ( 2.0.0 ).

For now you have to use this syntax ( with DateValue ) : 

```java
Assertions.assertThat(someTable).row(0).value("date_column").isEqualTo(DateValue.of(2020, 4, 6));
```

**Comment by VanRoy** [comment]
@rherrmann This enhancement has been released in the first RC of 2.0.0 today ( 2.0.0.RC1 ).

**Comment by VanRoy** [comment]
@rherrmann 2.0.0 is available now . Thanks for your patience.
